checks.d/elastic.py

```diff
@@ -158,6 +166,15 @@ class ESCheck(AgentCheck):
         "jvm.mem.non_heap_used": ("gauge", "jvm.mem.non_heap_used_in_bytes"),
         "jvm.threads.count": ("gauge", "jvm.threads.count"),
         "jvm.threads.peak_count": ("gauge", "jvm.threads.peak_count"),
+        "elasticsearch.fs.total.total_in_bytes": ("gauge", "fs.total.total_in_bytes"),
+        "elasticsearch.fs.total.free_in_bytes": ("gauge", "fs.total.free_in_bytes"),
+        "elasticsearch.fs.total.available_in_bytes": ("gauge", "fs.total.available_in_bytes"),
+        "elasticsearch.fs.total.disk_reads": ("gauge", "fs.total.disk_reads"),
```

Comment:
If the value is continuously increasing (as i expect given the metric name) it makes more sense to send it as a rate. The agent would then compute a "rate per second" which would be way more actionable in a monitor or in a graph than a continuously increasing value. Feel free to ping me if you have more questions!